deepform/data/add_features.py

```diff
@@ -45,6 +45,15 @@ class TokenType(Enum):
     "gross_amount": dollar_similarity,
 }
 
+MAX_TOKENS_BY_TARGET = {
+    # Each label column, and the maximum expected tokens that it uses.
+    "contract_num": 3,
+    "advertiser": 11,
+    "flight_from": 3,
+    "flight_to": 3,
+    "gross_amount": 3,
+}
+
 
 def extend_and_write_docs(source_dir, manifest, pq_index, out_path, max_token_count):
```

Comment:
Oops, we should also be removing all of these
```suggestion
def extend_and_write_docs(source_dir, manifest, pq_index, out_path):
```